server/routes/overview/overviewController.ts

```diff
@@ -1,92 +1,26 @@
-import createError from 'http-errors'
 import { RequestHandler } from 'express'
-import type { Goal, PrisonerGoals } from 'viewModels'
-import { EducationAndWorkPlanService, InductionService } from '../../services'
-import { mostRecentFunctionalSkills } from '../functionalSkillsResolver'
-import logger from '../../../logger'
+import { EducationAndWorkPlanService } from '../../services'
 import OverviewView from './overviewView'
 
 export default class OverviewController {
-  constructor(
-    private readonly inductionService: InductionService,
-    private readonly educationAndWorkPlanService: EducationAndWorkPlanService,
-  ) {}
+  constructor(private readonly educationAndWorkPlanService: EducationAndWorkPlanService) {}
 
   getOverviewView: RequestHandler = async (req, res, next): Promise<void> => {
     const { prisonNumber } = req.params
-    const {
+    const { prisonerSummary, curiousInPrisonCourses, actionPlanReviews, prisonerFunctionalSkills, induction } =
```

Comment:
The controller now gets everything it needs from `res.locals`, which is all populated in the route definition in `index.ts` below